### DISCUSSION TITLE
error: no member named 'set_hdf5_writing' in 'libMesh::ExodusII_IO'

### DISCUSSION THREAD

**Discussion by aruybalid** [idaholab/moose#20978] [Q&A Getting Started] [answered]
Hi, 

I am trying to compile moose, version idaholab/next, but get the following error upon compile (from within tensor_mechanics module):
```
In file included from /Users/aruybalid/projects/moose/framework/build/unity_src/constraints_Unity.C:3:
/Users/aruybalid/projects/moose/framework/src/constraints/AutomaticMortarGeneration.C:728:32: error: no member named 'set_hdf5_writing' in 'libMesh::ExodusII_IO'
    mortar_segment_mesh_writer.set_hdf5_writing(false);
```
 
I have updated mamba (`mamba update --all`), and done `make clobberall`, before compile.

This seems to be a libmesh-related issue.

Any idea how I can resolve this to proceed?

**Comment by GiudGiud** [comment, marked as answer]
solved over slack
mamba update had failed due to connection
